docs/run/runtime-options-overview.mdx

```diff
@@ -0,0 +1,559 @@
+|                      | pec                      | max_overhead            |  `None`/ integer >1                                                                      | `100`                      |
+|                      |                          | noise_gain              |  `auto`/ float in the range `[0, 1)`                                                     | `auto`                    |
+|                      | pec_mitigation           |                         | `True`/`False`                                                                           | `False`                      |
+|                      | zne_mitigation           |                         | `True`/`False`                                                                                | `False`                     |
+|                      | zne                      | noise_factors           |   list of floats; each float >= 1                                                                   | `(1, 3, 5)`                 |
+|                      |                          | extrapolator            | `'exponential'`<br />`'linear'`<br />`'double_exponential'`<br />`'polynomial_degree_(1 <= k <= 7)'` | (`'exponential'`, `'linear'`) |
+| resilience_level     |                          |                         | `0`/`1`/`2`                                                                                     | `1`                         |
+| seed_estimator       |                          |                         |  integer                                                                                |     `None`               |
+| simulator            |                          |                         |                                                                                   |                     |
+|                      | noise_model              |                         |   [Qiskit Aer NoiseModel](../verify/building_noise_models), or its representation        | `None`                      |
+|                      | seed_simulator           |                         |   integer                                                                               | `None`                    |
+|                      | coupling_map             |                         |  List where each entry specifies a directed two-qubit interaction, for example: [[0, 1], [0, 3], [1, 2], [1, 5], [2, 5], [4, 1], [5, 3]]     | full connectivity                   |
+|                      | basis_gates              |                         |  Set of single- and two-qubit gates that provide a universal gates set, meaning that any quantum operation can be decomposed into those gates. Example: `['ecr', 'x', 'sx', 'rz']` | The set of all basis gates supported by [Qiskit Aer simulator](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html) |
```

Comment:
Rather than "choices", these two have basically the full description. Maybe for `coupling_map` we can say "List of directed two-qubit interaction". And for `basis_gates` "List of basis gate names to unroll to".